src/index.test.js

```diff
@@ -0,0 +1,50 @@
+
+const RegExpExtension = require('./index').re
+
+describe('Simple strings', ()=>{
+  test('"some word"', () => {
+    const re = new RegExpExtension()
+    const generatedRegExp = re.str('some word').get()
+    expect(generatedRegExp).toEqual(/some word/)
+  })
+
+  test('Another simple strings', ()=>{
+    const re = new RegExpExtension()
+    const generatedRegExp = re.str('another word').get()
+    expect(generatedRegExp).toEqual(/another word/)
+  })
+
+
+  describe('Special characters in .str() should be escaped', ()=>{
+    test('Wildcard (*)', ()=>{
```

Comment:
May be this test became redundant after adding another one.
As I see it, we lose nothing if remove this "narrower", more specific, test.